vyper/semantics/types/bytestrings.py

```diff
@@ -26,6 +26,10 @@ class _BytestringT(VyperType):
         is applied to a literal definition.
     """
 
+    # this is a carveout because the current syntax allows dynamic arrays of
+    # bytestrings, but not static arrays of bytestrings, and can be removed once
+    # the syntax for static arrays is changed from `int128[5]` to `Array[int128, 5]`
```

Comment:
since the syntax for static arrays may not change, i'd say:
```suggestion
    # this is a carveout because currently we allow dynamic arrays of
    # bytestrings, but not static arrays of bytestrings
```